packages/social-previews/src/facebook-preview/post/header/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { __ } from '@wordpress/i18n';
+import FacebookPostIcon from '../icons';
+import defaultAvatar from './default-avatar.png';
+import type { FacebookUser } from '../../types';
+
+import './styles.scss';
+
+const FacebookPostHeader: React.FC< { user?: FacebookUser } > = ( { user } ) => {
+	return (
+		<div className="facebook-preview__post-header">
+			<div className="facebook-preview__post-header-content">
+				<img
+					className="facebook-preview__post-header-avatar"
+					src={ user?.avatarUrl || defaultAvatar }
+					alt=""
+				/>
+				<div>
+					<div className="facebook-preview__post-header-name">
+						{ user?.displayName ||
+							// translators: name of a fictional Facebook User
+							__( 'Anonymous User' ) }
+					</div>
+					<div className="facebook-preview__post-header-share">
+						<span className="facebook-preview__post-header-time">
+							{
+								// translators: temporal indication of when a post was published
+								__( 'Just now' )
+							}
+						</span>
+						<span className="facebook-preview__post-header-dot" aria-hidden="true">
+							·
+						</span>
+						<FacebookPostIcon name="public" />
+					</div>
+				</div>
+			</div>
+			<ul className="facebook-preview__post-header-more">
+				<li></li>
+				<li></li>
+				<li></li>
+			</ul>
```

Comment:
You could try doing this with CSS and getting rid of all the extra markup, something like this:

```css
div {
  width: 24px;
  height: 4px;
  background-image: radial-gradient(circle at 2px 2px, black 2px, transparent 0);
  background-size: 10px 20px;
}
```

You can change the `10px` value to determine how close the dots are to each other.

Reply:
Good thinking!